index.bs

```diff
@@ -4,6 +4,8 @@ Shortname: picture-in-picture
 Level: 1
 Status: ED
 ED: https://w3c.github.io/picture-in-picture/
+TR: https://www.w3.org/TR/picture-in-picture/
+Previous Version: https://www.w3.org/TR/2020/WD-picture-in-picture-20200130/
```

Comment:
I wouldn't necessarily include a link to the previous published Working Draft in the Editor's Draft (that's the sort of thing that are tedious to maintain and may easily become wrong)

Reply:
I've just removed this line.